### PR TITLE
feat: Customize TAB button #841

### PR DESCRIPTION
**Pull request by Nurkambay**
Signed-off-by: Leonid Baranov <leonid.baranov@broadcom.com>

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -171,7 +171,29 @@
                     80
                 ]
             }
-        }
+        },
+        "keybindings": [
+            {
+                "key": "tab",
```
Comment (asatklichov) [inline clients/cobol-lsp-vscode-extension/package.json:177]: is it possible to do like 
"(editorLangId == cobol || editorLangId == COBOL ) && !inSnippetMode" ?

if yes, then we can remove 2 objects /duplications here

**Review by asatklichov — APPROVED** [review]
🍰

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (Nurkambay) [reply clients/cobol-lsp-vscode-extension/package.json:177]: Not working :-(

**Review by nalmabrcom — APPROVED** [review]
LGTM, great work!

**Comment by temanbrcom** [comment]
Please, change the commit message body to the "Customize TAB button" to let it start with a verb

**Review by temanbrcom — APPROVED** [review]
🔥

**Pull request merged**